css/styles.css

```diff
@@ -0,0 +1,159 @@
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+body#home {
```

Comment:
@jiniekwak when using a class or ID you can just can leave off the element that it is being applied to in the selector. Unless you find yourself in a situation where you need to make your selector more specific to apply or your style.